source/clear-linux/virtual-environments/kvm.rst

```diff
@@ -3,60 +3,80 @@
 Running Clear Linux as a KVM guest OS
 #####################################
 
-To run |CL| in a virtualized environment using abbr:`KVM (Kernel-based
-Virtual Machine)`, follow these steps.
+This section explains how to run |CLOSIA| in a virtualized environment using
+abbr:`KVM (Kernel-based Virtual Machine)`.
 
-1. Enable `Intel® Virtualization Technology`_ (Intel® VT) and
+Install KVM
+===========
+
+#. Enable `Intel® Virtualization Technology`_ (Intel® VT) and
    `Intel®Virtualization Technology for Directed I/O`_ (Intel® VT-d) in the
    host machine’s BIOS.
 
-2. Log in and get root privilege on the host machine::
+#. Log in and get root privilege on the host machine:
```

Comment:
could we said

#. Log in **open a text terminal** and get root privilege on the host machine:

?